Contact/Contact Class and Smaller Related Data Classes/PhysicalAddress.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class PhysicalAddress
```

Comment:
not sure what you are up to here as all of your properties are prrivate.

Reply:
This is an instance of me starting something but not fully finishing it. I was going to add public properties with get/set but apparently moved on and forgot to complete this.

Reply:
I have added the appropriate get/set statements and am leaving this with the intention of adding some rudimentary validation checks (as I try to keep code related to zip codes in a single class for now).